### DISCUSSION TITLE
V 14.2 has an error!  Scripting a slew to RA and DEC

### DISCUSSION THREAD

**Discussion by selseyastro** [Stellarium/stellarium#130] [Q&A] [answered]
Before reporting an issue, please *absolutely make sure* to check the closed issues 
(https://github.com/Stellarium/stellarium/issues?q=is%3Aissue+is%3Aclosed) 
whether it has been reported and solved/closed already! Don't report in this case.

Also look into the wiki (https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version) 
and check if you are attempting to report a known issue. Don't report in this case.

Also look into the User Guide before reporting unexpected but correct behaviour. Don't report in this case.

If you want to report a feature wish, delete these template lines above and below, and fill in. 

If you want to report a bug, please make sure to update and verify the bug still exists in the current version. 
In this case please also check current beta builds from http://stellarium.org
If bug still persists, delete text above down to and including this line and fill in:

### Expected Behaviour
scripted slew to RA /DEC

### Actual Behaviour
Slews 6 minutes low and right for each star so not on the ocular cross.

### Steps to reproduce

core.setDate('2018-04-01T22:00:00');
StelMovementMgr.zoomTo(2.0);
core.moveToRaDec('11h53m49s','+53d41m14s',1);
core.wait(10);
core.moveToRaDec('13h23m55s','+54d55m53s',1);
core.wait(10);

### System
* Stellarium version: <14.2>
* Operating system: <Windows 10>
* Graphics Card: <Manufacturer (likely Intel, NVidia, AMD?), Model (HD, Geforce, Radeon..., with model number), driver version?>

### Logfile 
Problem when scripting a slew to ra/dec.   
The star is not in the centre of the ocular.  It always seems to be about 6minutes off low and right.

Regards,
John Mallett

**Comment by gzotti** [comment, marked as answer]
http://stellarium.org/doc/head/classStelMainScriptAPI.html

In case you wanted to center on the stars, you must use core.moveToRaDecJ2000() if you feed J2000 coordinates.

**Comment by selseyastro** [comment]
Hey George,

I have not found a reference to that script command.

Ok I will give that a try.

Thanks
John Mallett
http://www.astro.me.uk

**Comment by selseyastro** [comment]
George,

That worked... So used to J2000 I assumed everything worked in J2000....wrong.. 
All the best
John
